src/QsCompiler/Optimizations/Utils/VariableRenaming.fs

```diff
@@ -91,6 +97,8 @@ and private VariableRenamingNamespaces(parent: VariableRenaming) =
         | QsTupleItem { VariableName = InvalidName } -> ()
         | QsTuple items -> Seq.iter processArgTuple items
 
+    override __.OnArgumentTuple argTuple = argTuple
```

Comment:
Just double checking: This is to say we _don't_ want to process elements within an argument tuple?

Reply:
We don't want to process argument tuples, yes.